### PR TITLE
Add some tests for <marked-github-behavior>

### PR DESCRIPTION
**Pull request by txsmith**
and SubstituteRenderedGitHubMarkdownBehavior

### CODE REVIEW

File: test/util/marked-github-element.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        }
+        mdElem.project = project;
+      });
+
+      test('passes markdown to substitutor', function(done) {
+        var markdown = 'testMD'
+        mdElem.createMarkdownSubstituter = function(p) {
+          console.log('create');
+          return function(err, text) {
+            assert.include(text, markdown);
```
Comment (TimvdLippe) [inline test/util/marked-github-element.html:45]: Why `include` instead of `equal`?

File: test/util/marked-github-element.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          assert.deepEqual(p, project);
+          done();
+        }
+        mdElem.project = project;
+      });
+
+      test('passes markdown to substitutor', function(done) {
+        var markdown = 'testMD'
+        mdElem.createMarkdownSubstituter = function(p) {
+          console.log('create');
```
Comment (TimvdLippe) [inline test/util/marked-github-element.html]: Leftover debug statement.

File: test/util/marked-github-element.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      });
+
+      test("replaces issue references with anchor", function() {
+        var anchor = createElements(substitutor('', issueReference))[0];
+        assert.equal(anchor.tagName, 'A');
+      });
+
+      test("replaces issue ref with anchor with owner in href", function() {
+        var anchor = createElements(substitutor('', issueReference))[0];
+        assert.include(anchor.href, project.owner);
```
Comment (TimvdLippe) [inline test/util/marked-github-element.html]: I would combine this with the test below to `assert.include(anchor.href, project.owner + '/' + project.name)` to enforce they are added in the correct order.

File: test/util/marked-github-element.html (hunk 1 above)
Comment (txsmith) [inline test/util/marked-github-element.html:45]: Because `text` is rendered markdown that includes HTML, and `markdown` is just a string

**Comment by TimvdLippe** [comment]
LGTM!

**Pull request merged**